homework-submission/todoapp/src/Components/AddForm.js

```diff
@@ -0,0 +1,61 @@
+import React, { Component } from 'react';
+import DatePicker from 'react-datepicker';
+import moment from 'moment';
+
+import 'react-datepicker/dist/react-datepicker.css';
+
+class AddTodo extends Component {
+  state = {
+    description: '',
+    deadline: '',
+    done: false,
+    startDate: moment()
+  }
+  handleTodoChange = (e) => {
+    this.setState({
+      [e.target.id]: e.target.value
```

Comment:
you are using the `id` here as the targeted attribute to specify the state property name, it's fine but while you are not using it inside of the Css part of the application.

And what if in some part you decided to change the `id` dynamically.

Perhaps it's a kind of better use for `name` attribute while the `id`s are on the input elements, and keeping the Css use for it's use?